apps/architect-web/src/templates/mental-health-networks.json

```
@@ -543,6 +543,11 @@
         "entity": "node",
         "type": "person"
       },
+      "behaviours": {
```

Biome (2.5.0)

[error] The key behaviours was already declared.

(lint/suspicious/noDuplicateObjectKeys)